integrations/server/test_covid_hosp.py

```diff
@@ -29,10 +29,16 @@ def setUp(self):
         cur.execute('truncate table covid_hosp_meta')
 
 
-  def insert_issue(self, cur, issue, value, record_type):
+  def insert_timeseries(self, cur, issue, value):
     so_many_nulls = ', '.join(['null'] * 57)
     cur.execute(f'''insert into covid_hosp_state_timeseries values (
-      0, {issue}, 'PA', 20201118, {value}, {so_many_nulls}, '{record_type}', {so_many_nulls}
+      0, {issue}, 'PA', 20201118, {value}, {so_many_nulls}, {so_many_nulls}
```

Comment:
if you change `57` to `114` two lines above, you can make this insert template clearer.  same goes for the near-identical method below.
```suggestion
      0, {issue}, 'PA', 20201118, {value}, {so_many_nulls}
```